docs/api/datetime.md

```diff
@@ -283,6 +286,22 @@ Developers can provide their own buttons for advanced custom behavior.
 
 <CustomizingButtons />
 
+## Highlighting Specific Dates
```

Comment:
Can we add some verbiage around when you might want to use an array versus a callback? (i.e. array is good when you have fixed dates, callback is good when you have repeating dates)